action.yml

```diff
@@ -163,14 +166,18 @@ runs:
         targets: ${{inputs.target}}
         components: ${{inputs.components}}
         override: ${{inputs.override}}
+        rust_toolchain_dir: ${{inputs.rust-toolchain-dir}}
       shell: bash
       run: |
-        if [[ -z "$toolchain" && ( -f "rust-toolchain" || -f "rust-toolchain.toml" ) ]]
+        if [[ -z "$toolchain" && ( -f "rust-toolchain" || -f "rust-toolchain.toml" || -f "$rust_toolchain_dir/rust-toolchain.toml") ]]
```

Comment:
The code should also check for the existence of `$rust_toolchain_dir/rust-toolchain` (without toml).

Ideally the check could also be combined, e.g., setting `$rust_toolchain_dir` to `./`.